package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@getflywheel/local-components",
-	"version": "13.3.3",
+	"version": "13.3.4",
```

Comment:
We've been holding off on version bumps in PR's and adding a separate commit for those at release time.

Reply:
https://github.com/getflywheel/flywheel-local/wiki/Publishing-local-components#bump-the-version-and-create-git-tag